osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -207,14 +212,14 @@ private void load(BeatmapManager beatmaps, AudioManager audio, DialogOverlay dia
             Carousel.BeatmapSets = this.beatmaps.GetAllUsableBeatmapSetsEnumerable();
 
             Beatmap.DisabledChanged += disabled => Carousel.AllowSelection = !disabled;
-            Beatmap.TriggerChange();
-
             Beatmap.ValueChanged += workingBeatmapChanged;
+
+            workingBeatmapChanged(Beatmap.Value);
```

Comment:
any reason this was changed?

Reply:
Because we don't have a `BindEventAndTrigger()` method of likewise. Pending further discussion.

Reply:
oh hey, this is actually regressing logic (the trigger change was seemingly done before the `ValueChanged` binding on purpose). will fix.